### DISCUSSION TITLE
Can it be called from .NET using IronPython nuget package? which python version it support?

### DISCUSSION THREAD

**Discussion by GH-Vijay** [ThilinaRajapakse/simpletransformers#1104] [Q&A]
IronPython nuget package for .NET only support python 2.7 version, so would like to know if SimpleTransformers.Classification would work with IronPython for calling python file from .NET web api. What version of python language simple transformers support?

**Comment by ThilinaRajapakse** [comment]
Simple Transformers does not support Python 2.
